plugins/plugin-svelte/plugin.js

```diff
@@ -97,7 +102,7 @@ module.exports = function plugin(snowpackConfig, pluginOptions = {}) {
       'svelte-hmr/runtime/hot-api-esm.js',
       'svelte-hmr/runtime/proxy-adapter-dom.js',
     ],
-    async load({filePath, isHmrEnabled, isSSR}) {
+    async load({filePath, isHmrEnabled, isSSR, isPackage}) {
       let codeToCompile = await fs.promises.readFile(filePath, 'utf-8');
```

Comment:
nit: `utf8` is the Node encoding; `utf-8` is an alias. While I think it doesn’t matter here, I think there’s one Node library (not `fs`) that doesn’t accept `utf-8` as an encoing; only `utf8`, and will cause issues (if only I could remember what it was, then this comment would be more helpful!).